### DISCUSSION TITLE
Valuesets Valneva

### DISCUSSION THREAD

**Discussion by kamianni** [Digitaler-Impfnachweis/certification-apis#246] [General]
Hallo Zusammen,

wie sind denn die Valuesets für den neuen Impfstoff "Valneva". Laut KBV wird dieser die Impfziffer 88336x belegen und ist ab dem 06. September bestellbar.

mp=VLA2001 (?)
ma=ORG-100036422
vp=J07BX03

als mp habe ich auf Seiten der European Commission folgenden Wert gefunden: EU/1/21/1624

Dieser findet sich allerdings in den vorhandenen Valuesets nicht wieder...
https://ec.europa.eu/health/documents/community-register/html/h1624.htm

Viele Grüße

**Comment by TH-Pega** [comment]
EU/1/21/1624 ist m.E. korrekt (vgl. https://www.pei.de/DE/arzneimittel/impfstoffe/covid-19/covid-19-node.html), allerdings lässt sich damit in der RU aktuell (noch?) kein Zertifikat erstellen ("Error Validating with Schema #/v/0/mp ...").

**Reply by kamianni** [reply]
Und klappt's mit VLA2001? Das ist ja aktuell im mp-Valueset enthalten :-)
Entspricht halt nicht dem üblichen Schema, weshalb ich lieber nochmal nachhake.

**Reply by TH-Pega** [reply]
Ja, damit lässt sich ein Zertifikat abrufen. Frage mich aber, ob bei Real-Kontrollen ein Nicht-EU-Produktcode evtl. zu Problemen führt. (Das überbürokratische Agieren des BMG während der Modul-Vertragslaufzeit bewirkte wenigstens klare Regeln.)

**Reply by mge-medisoftware** [reply]
Der Impfstoff Valneva ist durch Praxen ab nächste Woche bestellbar...
Soll nun EU-Produktcode "EU/1/21/1624" wie üblich oder "VLA2001" im Element "mp" in Produktivumgebung übertragen werden?

**Comment by kamianni** [comment]
Folgende Rückmeldung haben wir bekommen:

Guten Tag,

wir möchten Sie informieren, dass aktuell noch keine Aufnahme des neuen Impfstoffes Valneva in die Valuesets der EU erfolgt ist und diese im Zeitraum um den 14.09.2022 geplant ist.

Sobald uns das um Valneva aktualisierte Valueset von der EU vorliegt werden wir die entsprechenden Anpassungen im nächsten Release vornehmen.

Sobald es Neuigkeiten gibt werden wir Sie über die weiteren Schritte informieren.

Viele Grüße
Ihr UBIRCH DCC Support

**Comment by kamianni** [comment]
Impfzertifikate für Valneva sind mit folgenden Werten auf der Referenzumgebung testbar:
mp=EU/1/21/1624
ma=ORG-100036422
vp=J07BX03

**Comment by mwicke** [comment]
We have included valneva in the documentation now: https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/Implementation.md
